qutebrowser/browser/downloads.py

```diff
@@ -745,20 +733,31 @@ def fetch(self, reply, fileobj=None, filename=None, auto_remove=False):
         return download
 
     @cmdutils.register(instance='download-manager', scope='window')
-    def cancel_download(self, count: {'special': 'count'}=1):
-        """Cancel the first/[count]th download.
+    def download_cancel(self, count: {'special': 'count'}=0):
```

Comment:
Same here - please keep `cancel_download` around for now, with `deprecated="Use :download-cancel instead!"` or so.